internal/contour/virtualhost.go

```diff
@@ -145,3 +150,10 @@ func clusteraction(cluster string) *v2.Route_Route {
 		},
 	}
 }
+
+func sslRedirect(i *v1beta1.Ingress) bool {
+	if i.Annotations["kubenetes.io/ingress.ssl-redirect"] == "false" {
```

Comment:
this can be simplified to

   return i.Annotations["kubenetes.io/ingress.ssl-redirect"] == "true"